### DISCUSSION TITLE
tutorial for multiple equation regressions

### DISCUSSION THREAD

**Discussion by mnky9800n** [MilesCranmer/PySR#348] [Ideas]
As far as I can tell, across all of the tutorials there is a theme, every regression only occurs for a single equation. However, you can imagine that there are low dimensional problems that produce multiple equations of motion. For example, a double pendulum has two equations of motion since there are two masses to keep track of (see this notebook to implement a numerical simulation of a double pendulum):

https://github.com/thwbecker/ML-for-EQ-physics-discovery/blob/tutorial1/Module1.2_DoublePendulum.ipynb

However, I am kind of stuck in trying to get PySR to produce multiple equations. Are there some code examples I could start from? I am happy to produce full fledged tutorials for PySR if I can get this to work. There are other applications as well this could apply to. Happy to hear ideas about it.

**Comment by MilesCranmer** [comment]
The simplest way to get multiple equations is to simply use multiple columns in the `y` variable you pass to `model.fit()`. There will be a set of equations for each column.
